client/src/Components/Star/Star.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import { Story } from '@storybook/react';
+
+import StarRating from './StarRating';
+
+import { action } from '@storybook/addon-actions';
```

Comment:
To idzie do góry do drugiej linijki
Z tego co wiem, to jest tak że jako pierwsze importujemy jakby pliki z nodeModules a wiec właśnie takie pliki jak teraz w 1 i 5 linijce, a dopiero później pliki od nas z projektu czyli ze sciezka relatywna, tak jak zasugerowała Klara można zrobić przerwę miedzy tymi dwiema częściami, ale często nie jest to błędem(Ja np. nie robię najczęściej), ale zgodnie ze sztuką powinno się robić :D